addons/mail/tests/test_link_preview.py

```diff
@@ -111,6 +111,7 @@ def test_get_link_preview_from_url(self):
                 self.assertEqual(preview, expected)
 
     def test_link_preview(self):
+        self.env['bus.bus'].sudo().search([]).unlink()
```

Comment:
I think the fix would be ok, but suggesting to use `assertBus` instead

Reply:
Or just `self._reset_bus()` actually, `assertBus` doesn't really make a difference.